rotkehlchen/tests/api/test_tags.py

```diff
@@ -678,3 +683,69 @@ def test_delete_utilized_tag(rotkehlchen_api_server):
     assert len(query) == 1
     assert query[0][0] == f'{SupportedBlockchain.BITCOIN.value}{UNIT_BTC_ADDRESS1}'
     assert query[0][1] == 'public'
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [0])
+def test_delete_all_tags(rotkehlchen_api_server):
+    """Tests that trying to delete all remaining tags of a blockchain account works."""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    # Add a tag
+    with rotki.data.db.user_write() as cursor:
+        rotki.data.db.add_tag(
+            write_cursor=cursor,
+            name='public',
+            description='My public accounts',
+            background_color='ffffff',
+            foreground_color='000000',
+        )
+    # Add an account with a tag
+    btc_balances = ['10000']
+    setup = setup_balances(
+        rotki,
+        ethereum_accounts=None,
+        btc_accounts=[UNIT_BTC_ADDRESS1],
+        eth_balances=None,
+        token_balances=None,
+        btc_balances=btc_balances,
+    )
+    with setup.bitcoin_patch:
+        response = requests.put(
+            api_url_for(
+                rotkehlchen_api_server,
+                'blockchainsaccountsresource',
+                blockchain='BTC',
+            ), json={
+                'accounts': [{
+                    'address': UNIT_BTC_ADDRESS1,
+                    'label': 'my btc miner',
+                    'tags': ['public'],
+                }],
+            },
+        )
+    assert_proper_response(response)
+    # Now delete the tag from the account
```

Comment:
nitpick: You do not delete the tag. To trigger the error you are editing the account in order to give it new tags and make sure old tags are gone.

Please adjust this in a future PR. Also perhaps add two tags to that btc address, and in the edit, give it one. This would be more realistic a test.